Data_Dictionary.md

```diff
@@ -1,10 +1,22 @@
 # LAMP Data Exports
 
-Access instructions for all LAMP public data exports are available at [https://performancedata.mbta.com](https://performancedata.mbta.com). 
+LAMP provides public data exports to support transit performance analysis. [Access instructions for all LAMP public data exports](https://performancedata.mbta.com)\
+Some LAMP data exports are used by [OPMI](https://www.massdottracker.com/wp/about/what-is-opmi-2/) for Tableau dashboarding.
 
 LAMP currently produces the following sets of public data exports:
 - [Subway Performance Data](#subway-performance-data)
+- [Bus Performance Data](#bus-performance-data)
```

Comment:
Should be under "OMPI Tableau Exports" 

To match existing exports name would be something like "LAMP_(ALL/RECENT)_Bus_Events